configs/config/pretrain/vision_transformer/swav/swav_deit_b_2nodes.yaml

```diff
@@ -13,19 +12,18 @@ config:
     NUM_DATALOADER_WORKERS: 8
     TRAIN:
       DATA_SOURCES: [disk_folder]
-      #      DATA_PATHS: ["<path to train folder>"]
-      LABEL_SOURCES: [disk_folder]
-      DATASET_NAMES: [imagenet1k_debug_folder]
+      DATASET_NAMES: [imagenet1k_folder]
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.